batch/python/create_partitions.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+import json
+import os
+
+import click
+import psycopg2
+
+
+@click.command()
+@click.option("-d", "--dataset", type=str, help="Dataset name")
+@click.option("-v", "--version", type=str, help="Version name")
+@click.option("-p", "--partition_type", type=str, help="Partition type")
+@click.option("-P", "--partition_schema", type=str, help="Partition schema")
+def cli(dataset: str, version: str, partition_type: str, partition_schema: str) -> None:
+
+    click.echo(
+        f"python create_partition.py -d {dataset} -v {version} -p {partition_type} -P {partition_schema}"
```

Comment:
s/create_partition.py/create_partitions.py/

Reply:
Not quite sure what you mean

Reply:
It's only cosmetic, but you're echoing "create_partition.py" when it should be "create_partitions.py" (you missed an "s"). It's not that important.

Reply:
got it. Thanks for spotting